Formula/neix.rb

```diff
@@ -0,0 +1,22 @@
+class Neix < Formula
+  desc "RSS/Atom feed reader for your terminal"
+  homepage "https://github.com/tomschwarz/neix"
+  url "https://github.com/tomschwarz/neix/archive/refs/tags/v0.1.5.tar.gz"
+  sha256 "23d39f0695fe73a737725c8a1266e02a6a173dcaf0a649ba30db02bc3e54a42f"
+  license "GPL-3.0-only"
+
+  depends_on "cmake" => :build
+  uses_from_macos "curl"
+  uses_from_macos "ncurses"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    installed_version = "\e[0;33m[neix]\e[0m Installed version: v0.1.5\n"
+    assert_match installed_version, shell_output("#{bin}/neix -v")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thank you for the reply.

I could integrate the unit tests included in the `tests` binary, however I do have some difficulties to understand how to do this without any issues.

One solution, explained [here](https://github.com/tomschwarz/neix), is the configure the project with `cmake -DENABLE-TESTS=ON`, which results in two binaries: `neix` and `tests`.

However, I would have to copy manually the configuration files *and* `tests` binary before the `make install` command, in order to include them in the cellar directory, and include a

```ruby
test do
    system bin/"tests"
end
```

at the end.

Do you think `bin.install` or `prefix.install` a good solution for this?  
Unfortunately, I don't have other examples to "copy" stuff from - do you have any example to provide for this please?

Thanks in advance,

Reply:
We don't want to run unit tests or integration tests. We want to know if a user can do what they expect to with the app. 

If the app can validate an RSS feed, check the release feed for the current version for example.